src/openforms/submissions/tests/form_logic/test_modify_components.py

```diff
@@ -874,6 +876,55 @@ def test_component_visible_in_frontend(self):
             "Some data that must not be cleared!", submission_step.data["textField"]
         )
 
+    @tag("gh-2324")
+    @unittest.expectedFailure
+    def test_component_visible_with_date(self):
```

Comment:
do I understand it correctly from the commit message that this behaviour is currently broken in master too?

Reply:
check, just making sure we don't knowingly introduce a new bug!